roslibrust/src/tests/ros1_native_integration_tests.rs

```diff
@@ -4,7 +4,7 @@
 #[cfg(feature = "ros1_test")]
 mod tests {
     use log::*;
-    use roslibrust::ros1::{NodeError, NodeHandle};
+    use roslibrust_ros1::{NodeError, NodeHandle};
```

Comment:
Since integration tests will have similar visibility to internals as user code, I think it's probably best for the integration test to look like what we expect it to look like for users; i.e. keep `roslibrust::ros1::`

Reply:
Oh looking at this now, I see this is under `src` instead of `tests` which is where integration tests ought to go (since they have the same visibility into the crate as a user of the crate does).

Reply:
Good catch I'll re-org!